### PR TITLE
First draft V3C systems registration

### PR DESCRIPTION
**Pull request by laurilo**
As discussed at: http://mpegx.int-evry.fr/software/MPEG/Systems/PCC-SYS/V-PCC/-/issues/155 V3C systems group would like to register the four character codes in 23090-10. Please see the details in the pull request.

### CODE REVIEW

**Comment by podborski** [comment]
We didn't have chance to review it during MPEG#137. This PR will have some conflicts with https://github.com/mp4ra/mp4ra.github.io/pull/134 which we will address after https://github.com/mp4ra/mp4ra.github.io/pull/134 is merged.

**Comment by podborski** [comment]
@cconcolato I forgot to submit my review 🤦🏻 I did it now. But this is what I was referring to during the MPEG call.

File: CSV/sample-entries.csv [hunk 1]
```diff
@@ -38,6 +39,7 @@ dvav,AVC-based Dolby Vision derived from avc3,Video,Dolby Vision,
 dvh1,HEVC-based Dolby Vision derived from hvc1,Video,Dolby Vision,
 dvhe,HEVC-based Dolby Vision derived from hev1,Video,Dolby Vision,
 dyol,Dynamic overlay parameters,Metadata,OMAF,
+vyvm,Dynamic spatial region data,Volumetric visual media,V3C-SYS,
```
[the remark this answers is not in the document]
Comment (laurilo) [reply CSV/sample-entries.csv:42]: This is indeed an unfortunate typo. Should be `dyvm`.

File: CSV/sample-entries.csv [hunk 2]
```diff
@@ -140,6 +142,14 @@ tx3g,Timed Text stream,Text,3GPP,
 ulaw,Samples have been compressed using uLaw 2:1.,Audio,QT,
 unid,Dynamic Range Control (DRC) data,Metadata,DRC,
 urim,Binary timed metadata identified by URI,Metadata,ISO,
+v3a1,Multi-atlas multi-track V3C coding with atlas parameter sets only in sample entries,Volumetric visual media,V3C-SYS,
+v3ag,Multi-atlas multi-track V3C coding with atlas parameter sets in sample entries or samples,Volumetric visual media,V3C-SYS,
+v3c1,Single atlas multi-track V3C coding with atlas parameter sets only in sample entries,Volumetric visual media,V3C-SYS,
+v3cb,Multi-atlas multi-track V3C coding with common atlas data,Volumetric visual media,V3C-SYS,
```
[the remark this answers is not in the document]
Comment (laurilo) [reply CSV/sample-entries.csv:148]: I was struggling with the description here, not all the tracks are base tracks, such as the `v3a1` and `v3ag`. Maybe we can be less descriptive? Would something like below be more appropriate?

`v3a1, V3C atlas track with atlas parameter sets only in sample entries,Volumetric visual media,V3C-SYS`

File: CSV/sample-entries.csv [hunk 3]
```diff
@@ -140,6 +142,14 @@ tx3g,Timed Text stream,Text,3GPP,
...
+v3cg,Single atlas multi-track V3C coding with atlas parameter sets in sample entries or samples,Volumetric visual media,V3C-SYS,
+v3e1,Single-track V3C coding with atlas parameter sets only in sample entries,Volumetric visual media,V3C-SYS,
+v3eg,Single-track V3C coding with atlas parameter sets in sample entries or samples,Volumetric visual media,V3C-SYS,
+v3t1,Multi-track V3C coding with atlas tile data,Volumetric visual media,V3C-SYS,
```
[the remark this answers is not in the document]
Comment (laurilo) [reply CSV/sample-entries.csv:152]: Agree.

File: CSV/sample-entries.csv [hunk 4]
```diff
@@ -140,6 +142,14 @@ tx3g,Timed Text stream,Text,3GPP,
 ulaw,Samples have been compressed using uLaw 2:1.,Audio,QT,
 unid,Dynamic Range Control (DRC) data,Metadata,DRC,
 urim,Binary timed metadata identified by URI,Metadata,ISO,
+v3a1,Multi-atlas multi-track V3C coding with atlas parameter sets only in sample entries,Volumetric visual media,V3C-SYS,
```
[the remark this answers is not in the document]
Comment (laurilo) [reply CSV/sample-entries.csv:145]: Agree.

File: CSV/sample-entries.csv [hunk 5]
```diff
@@ -140,6 +142,14 @@ tx3g,Timed Text stream,Text,3GPP,
...
+v3ag,Multi-atlas multi-track V3C coding with atlas parameter sets in sample entries or samples,Volumetric visual media,V3C-SYS,
+v3c1,Single atlas multi-track V3C coding with atlas parameter sets only in sample entries,Volumetric visual media,V3C-SYS,
+v3cb,Multi-atlas multi-track V3C coding with common atlas data,Volumetric visual media,V3C-SYS,
+v3cg,Single atlas multi-track V3C coding with atlas parameter sets in sample entries or samples,Volumetric visual media,V3C-SYS,
+v3e1,Single-track V3C coding with atlas parameter sets only in sample entries,Volumetric visual media,V3C-SYS,
```
[the remark this answers is not in the document]
Comment (laurilo) [reply CSV/sample-entries.csv:150]: Agree.

File: CSV/sample-entries-boxes.csv [hunk 6]
```diff
@@ -61,6 +62,8 @@ udi2,Unified DRC instructions,Audio,DRC
 ueqc,Equalization coefficients,Audio,DRC
 ueqi,Equalization instructions,Audio,DRC
 uriI,URI Information,Metadata,ISO
+v3cC,V3C Configuration,Volumetric visual media,V3C-SYS
+v3tC,V3C Tile Configuration,Volumetric visual media,V3C-SYS
```
[the remark this answers is not in the document]
Comment (laurilo) [reply CSV/sample-entries-boxes.csv:66]: Agree.

File: CSV/item-properties.csv [hunk 7]
```diff
@@ -30,7 +30,10 @@ rwpk,Region-wise packing,OMAF
 stvi,Frame packing,OMAF
 subs,Subsample information,HEIF
 tols,Layered HEVC Target output layer set property,HEIF
+v3cC,V3C configuration,V3C-SYS
+v3tp,V3C atlas tile configuration,V3C-SYS
```
[the remark this answers is not in the document]
Comment (laurilo) [reply CSV/item-properties.csv:34]: `v3tC` and `v3tp` for some reason have different syntax in 23090-10, that is why in FDAM the v3tp was not changed.

**Review by podborski — APPROVED** [review]

**Pull request merged**